packages/ui/src/components/playback/Playback.module.scss

```
@@ -18,3 +18,33 @@
     width: 100px;
   }
 }
+
+.volumeTrigger {
+  position: relative;
+}
+
+.volumeMargin {
+  position: absolute;
+  bottom: 24px;
+  left: -12px;
+  width: 48px;
+  height: 180px;
+  display: none;
+}
+
+.volume {
+  position: absolute;
+  top: 12px;
+  left: 12px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 24px;
+  height: 144px;
+  border-radius: var(--radius);
+  background-color: var(--surface-color);
+}
+
+.volumeTrigger:hover .volumeMargin {
+  display: grid;
+}
```

Many hardcoded sizes here could be removed to let the container expand to whatever size the slider wants.
I'd also stick to the `24px` width. I'm assuming that these `48px` are meant to make the hover hitbox bigger but I'm not sure if that's necessary.
```suggestion
.volumeMargin {
  position: absolute;
  bottom: 24px;
  left: 0;
  right: 0;
  padding-bottom: 18px;
  display: none;
}

.volume {
  padding: 12px 0;
  border-radius: var(--radius);
  background-color: var(--surface-color);
}

.volumeTrigger:hover .volumeMargin {
  display: block;
}
```